Transfer/Sources/ScheduleTransferSectionData.swift

```diff
@@ -97,7 +100,7 @@ struct ScheduleTransferSummaryData: ScheduleTransferSectionData {
         if let destinationAmount = transfer.destinationAmount,
             let destinationCurrency = transfer.destinationCurrency {
             let destinationAmountFormattedString = destinationAmount.format(with: destinationCurrency)
-            guard let feeAmount = transfer.destinationFeeAmount else {
+            guard let feeAmount = transfer.destinationFeeAmount, Double(feeAmount) != 0 else {
                 rows.append((title: "transfer_amount_confirmation".localized(),
                              value: destinationAmountFormattedString))
```

Comment:
No need to format this as per my previous review comment, so use destinationAmount instead

Reply:
Remove the below line as well. As per ticket when there's no fee do not show fee and You will Receive(transfer_net_amount_confirmation) cell. Only transfer_amount_confirmation will be shown

Reply:
@sarun-altmtrk Kindly fix the above comment. `No need to format this as per my previous review comment, so use destinationAmount instead`